jvs.js

```diff
@@ -0,0 +1,79 @@
+import Store from './modules/store.js';
+import UI from './modules/ui.js';
+
+class Book {
+  constructor(title, author, id = Math.floor(Math.random() * 1000000000000000)) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+document.addEventListener('DOMContentLoaded', UI.showBooks);
+document.querySelector('#book-card').addEventListener('submit', (e) => {
+  e.preventDefault();
+  // console.log('hey');
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  if (title !== '' && author !== '') {
+    const book = new Book(title, author);
+    UI.addBookToList(book);
+    UI.clearFields();
+    Store.addBooks(book);
+  }
+});
+
+document.querySelector('.bookDisplay').addEventListener('click', (e) => {
+  if (e.target.className === 'remove') {
+    const id = e.target.previousElementSibling.innerText;
+    Store.deleteBookStorage(id);
+    UI.deleteBook(e.target);
+  }
+});
+
+// const title = document.querySelector('#title');
+// const author = document.querySelector('#author');
+// const form = document.getElementById('book-card');
+// const library = document.getElementById('library');
+// let book;
+// let bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+// function addBook() {
+//   book = {
+//     title: title.value,
+//     author: author.value,
+//     id: Math.floor(Math.random() * 1000000),
+//   };
+//   bookList.push(book);
+//   localStorage.setItem('bookList', JSON.stringify(bookList));
```

Comment:
- [ ] Please remove commented code from your codebase as it makes your work look untidy 👍